code/game/turfs/simulated/wall_attacks.dm

```diff
@@ -228,7 +228,7 @@
 			if(cut_delay<0)
 				cut_delay = 0
 
-			if(!do_after(user,cut_delay))
+			if(!do_after(user,4/W.toolspeed))

```

Comment:
Each material has it's own delay depending on how strong it is, this should be "cut_delay/W.toolspeed"
and the above line should be set to 1 to avoid dividing by 0